### PR TITLE
UX: update topic small action post to add link for new linked topic URL

### PR DESCRIPTION
**Pull request by techAPJ**
https://meta.discourse.org/t/linked-topics-splitting-and-managing-megatopics/168992/4?u=techapj

### CODE REVIEW

**Comment by gschlager** [comment]
Should the following be done as well? From https://meta.discourse.org/t/168992/3

> I’d recommend replacing [title](url) with just url, and relying on existing Discourse functionality that displays the title of the topic from the URL. If I’m not mistaken, this would have the bonus of auto-updating the topic title displayed if the topic is renamed, which I think would be a nice touch.

File: spec/jobs/create_linked_topic_spec.rb [hunk 1]
```diff
@@ -44,7 +44,6 @@
       new_topic = Topic.last
       linked_topic = new_topic.linked_topic
       expect(topic.title).to include(I18n.t("create_linked_topic.topic_title_with_sequence", topic_title: raw_title, count: 1))
-      expect(topic.posts.last.raw).to eq(I18n.t('create_linked_topic.moderator_post_raw', new_url: new_topic.url))
```
Comment (gschlager) [inline spec/jobs/create_linked_topic_spec.rb:47]: Should there be a different expectation instead?

**Comment by techAPJ** [comment]
> Should the following be done as well? From https://meta.discourse.org/t/168992/3

I tried doing that here but the link is not oneboxing... that is probably because the post processing does not happen for small action posts.

File: spec/jobs/create_linked_topic_spec.rb (hunk 1 above)
Comment (techAPJ) [reply spec/jobs/create_linked_topic_spec.rb:47]: Added it back. :)

**Review by gschlager — APPROVED** [review]

**Pull request merged**